packages/core/components/EmptyFileListMessage/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function EmptyFileListMessage() {
                     >
                         &nbsp;GitHub&nbsp;
                     </a>
-                    if you still expect there to be matches present.
+                    if you expect there should be matches present for query.
```

Comment:
nit: maybe 'for this query'?

Reply:
oof will there ever be a day I don't have a typo